qt/component.cpp

```diff
@@ -385,6 +386,19 @@ Bundle Component::bundle(Bundle::Kind kind) const
     return Bundle(bundle);
 }
 
+QList<AppStream::Suggested> AppStream::Component::suggested() const
+{
+    QList<Suggested> res;
+
+    auto suggestions = as_component_get_bundles(m_cpt);
```

Comment:
This is certainly wrong and will explode when converting it to AsSuggested later...